source/04-1_alignak_configuration/core_configuration_tips.rst

```diff
@@ -1,4 +1,4 @@
-.. _configuration/main:
+.. _configuration/core_tips:
 
 =============================================
 Main configuration file (alignak.cfg) options
```

Comment:
I open this issue (https://github.com/Alignak-monitoring/alignak-doc/issues/22) to make a review of all those parameters because I know some of them are not used or deprecated (old Nagios parameters)